packages/babel-plugin-ima-logger/jest.config.js

```diff
@@ -0,0 +1,15 @@
+const base = require("../../jest.config.base.js");
+
+module.exports = {
+    ...base,
+    globals: {
+        functions: 10,
+        lines: 10,
+        statements: 10
+    },
+    testEnvironment: "node",
+    setupFiles: [],
+    transform: {
+		"\\.jsx?": "<rootDir>/../../preprocess.js"
+	},
+};

```

Comment:
Well, all keys in this cvonfig should be in the root config.

Reply:
There's a problem with "transform" value if we run test in root or in package. The path of preprocess.js is different in both cases.

Reply:
See comment in root `jest.conf.js` file, if you remove the `...base` from there, you will be able to set it in the base config for all packages.

Reply:
Option `transform` is set globaly in `jest.config.base.js` file.